src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/statistics/StatisticsService.java

```diff
@@ -168,11 +168,10 @@ private void gatherPortStats(IOFSwitch iofSwitch) {
                 .setPortNo(OFPort.ANY)
                 .build();
 
-        logger.trace("Getting port stats for switch={}", iofSwitch.getId());
+        logger.info("Getting port stats for switch={} OF-xid:{}", iofSwitch.getId(), portStatsRequest.getXid());
```

Comment:
could you please confirm that this xid is  already set, the same question is valid for all requests from this StatisticsService